README.md

```diff
@@ -124,58 +136,137 @@ export type PluginOptions = {
 
 ### rules
 
-An array of rules consisting of these properties:
+The rules that define whether a visit will be considered a fragment visit. Each rule consists of
+mandatory `from` and `to` URL patterns, an array `fragments` of selectors, as well as an optional
+`name` of this rule to allow precise styling. The from/to patterns are converted to regular
+expressions by [path-to-regexp](https://www.npmjs.com/package/path-to-regexp).
+
+```js
+{
+  rules: [
+    {
+      from: '/users/:filter?',
+      to: '/users/:filter?',
+      fragments: ['#users'],
+      name: 'list'
+    }
+  ]
+}
+```
+
+### (List)
 
-#### from (required)
+- `from`, required, `string | string[]` — The pattern(s) to match against the previous URL
```

Comment:
Two times "optional"

Reply:
Not sure what this is referring to? Maybe wrong line marked?

Reply:
I think this was already fixed in one of your commits